common/source/docs/common-compass-calibration-in-mission-planner.rst

```diff
@@ -22,30 +22,25 @@ Calibration first steps
 .. warning:: Do not calibrate the compasses near any metallic or magnetic field producing object (computers, cell phones, metal desks, power supplies, etc.) or incorrect calibration will occur.
 
 
-- Under **Initial Setup \| Mandatory Hardware** select **Compass**.
+- Under **SETUP\| Mandatory Hardware** select **Compass**.
 
-   .. figure:: ../../../images/MissionPlanner_CompassCalibration_MainScreen.png
-      :target: ../_images/MissionPlanner_CompassCalibration_MainScreen.png
+   .. figure:: ../../../images/CompassCalibration_Onboard.png
+      :target: ../_images/CompassCalibration_Onboard.png
 
       Mission Planner: Compass Calibration
 
-   You normally shouldn't need to change any of the "General Compass Settings" or compass specific values (i.e. "Compass #1" section), but you might want to confirm that the \ **Enable compasses** and **Obtain declination automatically** boxes are checked.
-
-   You may wish to disable any internal compasses if you are consistently seeing the "inconsistent compasses" pre-arm message often and you are sure that the external compass is calibrated and the orientation is correct.
+   You may wish to disable any internal compasses if you are consistently seeing the "inconsistent compasses" pre-arm message often and you are sure that the external compass is calibrated. 
 
 .. _onboard_calibration:
 
 Onboard Calibration
 ===================
 
-Copter-3.4, Plane 3.4.0, and Rover 3.0.0 (and higher) support "Onboard Calibration" meaning that the calibration routine runs on the autopilot.  This method is more accurate than the older "Offboard Calibration" (aka "Live Calibration") which runs on the ground station because in addition to offsets, scaling is also calculated.
+"Onboard Calibration" is a calibration routine that runs on the autopilot.  This method is more accurate than the older "Offboard Calibration" (aka "Live Calibration") which runs on the ground station because in addition to offsets, scaling and orientation is also automatically determined.
```

Comment:
I think it should be "**are** also automatically determined".

Reply:
yep